src/test/java/com/team3/central/services/CategoriesServiceTest.java

```diff
@@ -0,0 +1,89 @@
+package com.team3.central.services;
+
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import com.team3.central.repositories.CategoryRepository;
+import com.team3.central.repositories.entities.Category;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class CategoriesServiceTest {
+  private CategoryRepository mockCategoryRepository;
+  private CategoriesService categoriesService;
+
+  @BeforeEach
+  void setUp() {
+    mockCategoryRepository = Mockito.mock(CategoryRepository.class);
+    categoriesService = new CategoriesService(mockCategoryRepository);
+  }
```

Comment:
Can't we user @BeforeAll with static void setup(), or is it necessary to do it before each test ?